pkg/console/operator/sync_v400.go

```diff
@@ -35,6 +37,21 @@ import (
 	servicesub "github.com/openshift/console-operator/pkg/console/subresource/service"
 )
 
+var (
+	consoleURLMetric = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
A `guage` can go up and down.  We are using it as a boolean, really, `1`, for `have a url`, `0` for `don't have a url`.

Reply:
if set to `0` the `url` will be `nil` ?